packages/venia-ui/lib/components/CommunicationsPage/communicationsPage.css

```diff
@@ -0,0 +1,50 @@
+.root {
+    padding: 20px;
```

Comment:
Preference for rems over pixel values when possible:

```suggestion
    padding: 1.25rem;
```